uber/site_sections/signups.py

```diff
@@ -60,6 +61,23 @@ def shirt_size(self, session, message='', shirt=None, second_shirt=None, csrf_to
             'opts': [('', 'Enter your shirt size')] + c.SHIRT_OPTS[1:]
         }
 
+    @check_shutdown
+    def volunteer_agreement(self, session, message='', agreed_to_terms=None, csrf_token=None):
+        attendee = session.logged_in_volunteer()
+        if csrf_token is not None:
```

Comment:
Our usual tactic is to check if the request was POST, rather than checking if a csrf token was in the request. I wonder which is better? 🤔

Reply:
I think checking for POST is generally better.